plugins/Goals/API.php

```diff
@@ -507,7 +504,7 @@ public function getVisitsUntilConversion($idSite, $period, $date, $segment = fal
         $dataTable = $this->getGoalSpecificDataTable(
             Archiver::VISITS_UNTIL_RECORD_NAME, $idSite, $period, $date, $segment, $idGoal);
 
-        $dataTable->queueFilter('Sort', array('label', 'asc', true));
+        $dataTable->queueFilter('Sort', array('label', 'asc', true, false));
```

Comment:
I do not want to change those kinda Sort filters for now as it is risky to break something.